cmake/Gettext.cmake

```diff
@@ -0,0 +1,83 @@
+set(LANGUAGES
```

Comment:
You do not need to fill the language list manually. Each `po` folder has a `LINGUAS` file. Read the list of languages using the [file](https://cmake.org/cmake/help/v3.0/command/file.html) function.